src/state/metadata/logics.ts

```diff
@@ -219,9 +222,64 @@ const loadLocalFile = createLogic({
     type: LOAD_LOCAL_FILE_IN_VIEWER,
 });
 
+const loadFileViaUrl = createLogic({
+    process(deps: ReduxLogicDeps, dispatch, done) {
+        const { action, getState } = deps;
+        const url = action.payload;
+        const currentState = getState();
+        dispatch(
+            setViewerStatus({
+                status: VIEWER_LOADING,
+            })
+        );
+        let simulariumController = getSimulariumController(currentState);
+        if (!simulariumController) {
+            if (action.controller) {
+                simulariumController = action.controller;
+                dispatch(setSimulariumController(simulariumController));
+            }
+        }
```

Comment:
Can it ever happen that (!simulariumController) and (!action.controller) ?

Reply:
this comment: https://github.com/allen-cell-animated/simularium-website/blob/master/src/state/metadata/logics.ts#L4

Reply:
Ah... yeah... I vaguely remember trying to see if removing Mocha would solve the problem but it didn't solve it. So unless it magically fixed itself it is still broken.